app/controllers/articles_controller.rb

```diff
@@ -41,10 +65,30 @@ def create
   # PATCH/PUT /articles/1
   # PATCH/PUT /articles/1.json
   def update
+    img = params[:article][:img]
```

Comment:
Quando o usuário for editar, poderia aparecer a imagem atribuída anteriormente aquele artigo